packages/ui/dialogs/add-debt/index.tsx

```diff
@@ -119,6 +121,17 @@ class AddDebt extends Component<Props, State> {
     }
   }
 
+  setFriend(friend: Friend) {
+    const { hasPendingMessage, getPendingFromFriend } = this.props
+    const { route, pendingTransaction, pendingSettlement } = getPendingFromFriend(friend.nickname)
+    if(route) {
+      this.props.navigation.navigate(route, { pendingSettlement, pendingTransaction })
+      hasPendingMessage()
```

Comment:
I am not convinced we need to throw the Toast error here, although I could go either way. At this point, the user has tapped on the red `Pending` button so they would probably expect to be taken to that Pending transaction. 

If they tap on the entire row (as if intending to select that friend for a new transaction), then there's a stronger argument for displaying the error.

Reply:
Maybe having it in red isn't great, but I like the idea of having a notification just to make it clear that they can't make a new transaction without resolving the existing one